### ISSUE TITLE
No way to specify empty environment variables

### ISSUE THREAD

**Issue by joshwget** [docker/libcompose#381]
There isn't currently a way to specify empty environment variables for a service. For the following template, I'd expect `TEST` to appear as an empty environment variable.

```
test:
  image: ubuntu
  command: env
  environment:
    TEST: ""
```
```
$ ./bundles/libcompose-cli up                   
test_1 | PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
test_1 | HOSTNAME=c5f0ab2fdd94
test_1 | HOME=/root
```

**Comment by vdemeester** [comment]
@joshwget sounds fair, `docker/compose` creates an empty enviornment variable so this should be considered as a bug.

**Comment by joshwget** [comment]
Fixed by #384
